swap_meet/decor.py

```diff
@@ -0,0 +1,12 @@
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, condition = 0):
+        self.category = "Decor"
+        self.condition = float(condition)
```

Comment:
Another way to write this is to use super to take advantage of the code in Item's constructor:
```
def __init__(self, condition = 0):
    super().__init__("Decor", condition)
```